components/brave_federated_learning/DEPS

```diff
@@ -0,0 +1,14 @@
+include_rules = [
+  "+bat/ads",
+  "+brave/components/version_info",
+  "+content/public/browser",
+  "+services/network/public",
+  "+third_party/metrics_proto",
+]
+
+# Existing exceptions
+specific_include_rules = {
+  "brave_federated_learning_service.cc": [
+    "+chrome/browser/profiles/profile_manager.h",
```

Comment:
don't do that, we must not add any new exceptions. If it really needs the profile manager, it should belong to `browser`, not components, or have a delegate, implemented on the browser side

Reply:
removed it 👍